locales/en/tags.json

```diff
@@ -1,4 +1,8 @@
 [
+	{
+		"tag.67.description": "Designed specifically to block ads on web pages in the Hindi language",
```

Comment:
Without Bengali and Tamil here?

Reply:
yes, one tag per one language, this one about Hindi

Reply:
Yep, that was on purpose like @slavaleleka said.

Reply:
if we mean more than the Hindi language here then we could also write "in the Indic languages"